src/server.ts

```diff
@@ -78,14 +80,20 @@ export class JSONRPCServer<ServerParams = void> {
     this.addMethodAdvanced(name, this.toJSONRPCMethod(method));
   }
 
+  setDefaultMethod(method: SimpleJSONRPCMethod<ServerParams>): void {
```

Comment:
Please rename it to `addDefaultMethod` to be consistent with existing API:
```suggestion
  addDefaultMethod(method: SimpleJSONRPCMethod<ServerParams>): void {
```

Reply:
Oh, actually `set` makes sense because it only has one default method. Never mind!